projects/lib/package.json

```diff
@@ -9,5 +9,6 @@
     "dependencies": {
         "deepmerge": "^4.2.2",
         "tslib": "^2.3.0"
-    }
+    },
+    "license": "MIT"
```

Comment:
The MIT license was specified in the main package.json, and was the license displayed on the npm registry in previous versions.